packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -132,10 +132,11 @@ export const start = async () => {
         locals
     })
 
+    const routes = await getAllRoutes(locals)
```

Comment:
Sync up with Jinsu to understand his requirements on seo extension, we'll have to pass in the location/req for example. It might already be in locals, but I'm not sure. We should experiment with passing in the req maybe to see how that looks

Reply:
You might want to pick his brain about the deserialization/serialization of extensions to make sure it will fit in to what we have here.